app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>Here are the list of users</h1>
\ No newline at end of file
+<h1>users show</h1>
+
+<div>
+  <h1>Here are the list of users</h1>
+  <ul>
+    <% @users.each_with_index do |user|%>
+      <li>
+        <%= image_tag "#{user.photo}" %>
+        <div>
+            <%=  %>
+            <%= link_to "#{user.name}", user_path(user.id) %>
+            <span>Number of posts: <%= user.posts_counter || 0 %></span>
+       </div>
+      </li> 
+    <%end%>
```

Comment:
- [ ] Please consider fixing your counters, I created `7` posts but it shows `21`:

![image](https://github.com/Reggeditt/blog-rails/assets/77069011/e0b9da33-bbbd-4921-a753-7877c62d9c68)

![image](https://github.com/Reggeditt/blog-rails/assets/77069011/6a07ac5c-9bc1-4f99-93fe-e6105b5cf442)


- [ ] **[OPTIONAL]** I suggest you use `each` instead of `each_with_index` if you're not using the index, like in this case. 🤓 🔧 

- [ ] **[OPTIONAL]** Also, If you're interested in going one step further, you could use [`Partials`](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) to display users:

```rb
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```

It would be great if you implemented this! 🤓 🔧

Reply:
fixed